extensions/amp-bind/amp-bind.md

```diff
@@ -73,10 +71,21 @@ The implicit state can be initialized at page load with custom JSON. For example
 </amp-state>
 ```
 
-Binding expressions can reference the data in this `<amp-state>` component via dot syntax.
-In this example, `myState.foo` will evaluate to `"bar"`.
+[Expressions](#expressions) can reference state variables via dot syntax. In this example, `myState.foo` will evaluate to `"bar"`.
+
+
+##### AMP.setState()
+
+State can be mutated by the new `AMP.setState()` [action](../../spec/amp-actions-and-events.md).
 
-## Data binding
+- `AMP.setState()` performs a *shallow merge* of its arguments with the document state.
+- `AMP.setState()` can override data initialized by `amp-state`.
+
+### Binding
+
+A binding is a special attribute of the form `[property]` that links an element's property to an [expression](#expressions).
```

Comment:
Wanted to differentiate between the attribute that includes the brackets and property within the brackets.